pkgs/development/web/nodejs/gyp-patches.nix

```diff
@@ -0,0 +1,22 @@
+{ fetchpatch }:
+let
+  name = "gyp-darwin-sandbox.patch";
+  url = "https://web.archive.org/web/20240506065753/https://github.com/nodejs/gyp-next/pull/216.patch";
```

Comment:
```suggestion
  url = "https://github.com/nodejs/gyp-next/commit/706d04aba5bd18f311dc56f84720e99f64c73466.patch";
```

Reply:
Changed to the [PR patch URL](https://github.com/nodejs/gyp-next/pull/216.patch) that AFAIK has stable content after merge. Also updated to use `fetchpatch2`.

Reply:
Let's use the commit URL instead, the PR might get deleted or whatnot, and the actual commit that landed is what we should be fetching anyway.